src/vs/workbench/contrib/notebook/browser/controller/executeActions.ts

```diff
@@ -488,22 +489,22 @@ registerAction2(class ExecuteCellInsertBelow extends NotebookCellAction {
 	}
 });
 
-registerAction2(class CancelNotebook extends NotebookAction {
+class CancelNotebook extends NotebookAction {
+	override getEditorContextFromArgsOrActive(accessor: ServicesAccessor, context?: UriComponents): INotebookActionContext | undefined {
+		return getContextFromUri(accessor, context) ?? getContextFromActiveEditor(accessor.get(IEditorService));
+	}
+
+	async runWithContext(accessor: ServicesAccessor, context: INotebookActionContext): Promise<void> {
+		return context.notebookEditor.cancelNotebookCells();
+	}
+}
+
+registerAction2(class CancelAllNotebook extends CancelNotebook {
 	constructor() {
 		super({
 			id: CANCEL_NOTEBOOK_COMMAND_ID,
 			title: localize('notebookActions.cancelNotebook', "Stop Execution"),
```

Comment:
Is this exposed in the Command Palette? If so can you use an `ILocalizedString`?

Reply:
@TylerLeonhardt good catch, I should use `ILocalizedString` instead of this legacy style.